tools/ios/build_xcframework.py

```diff
@@ -50,38 +65,49 @@ def build_framework_for_platform_and_arch(
     config: str,
     opencv_dir: Path,
     ios_deployment_target: str,
+    macos_deployment_target: str,
     other_build_args: list[str],
-):
-    cmake_defines = [
-        # required by OpenCV CMake toolchain file
-        # https://github.com/opencv/opencv/blob/4223495e6cd67011f86b8ecd9be1fa105018f3b1/platforms/ios/cmake/Toolchains/common-ios-toolchain.cmake#L64-L66
-        f"IOS_ARCH={arch}",
-        # required by OpenCV CMake toolchain file
-        # https://github.com/opencv/opencv/blob/4223495e6cd67011f86b8ecd9be1fa105018f3b1/platforms/ios/cmake/Toolchains/common-ios-toolchain.cmake#L96-L101
-        f"IPHONEOS_DEPLOYMENT_TARGET={ios_deployment_target}",
-    ]
-
+):  
     build_cmd = (
         [
             sys.executable,
-            str(_repo_dir / "tools" / "build.py"),
-            f"--build_dir={build_dir}",
-            f"--config={config}",
-            "--update",
-            "--build",
-            "--parallel",
-            "--test",
-            # iOS options
-            "--ios",
-            "--build_apple_framework",
-            f"--ios_sysroot={platform}",
-            f"--ios_toolchain_file={_get_opencv_toolchain_file(platform, opencv_dir)}",
-            f"--apple_arch={arch}",
-            f"--apple_deploy_target={ios_deployment_target}",
+                str(_repo_dir / "tools" / "build.py"),
+                f"--build_dir={build_dir}",
+                f"--config={config}",
+                "--update",
+                "--build",
+                "--parallel",
+                "--test",
+                "--build_apple_framework",
+                f"--apple_sysroot={platform}",
+                f"--apple_arch={arch}",
```

Comment:
nit: indentation looks off. would have expected it to align with the start of `sys.executable` given these are all items in the same array.